src/styles/card.scss

```
@@ -0,0 +1,82 @@
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__picture {
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
+    font-size: $mainElementsSize;
+    font-weight: 500;
+    line-height: $mainLineHeight;
+
+    @include block-distance;
+  }
+
+  &__product-code {
+    font-size: $secondaryElementsSize;
+    line-height: $secondaryLineHeight;
+    color: $secondaryColour;
+
+    @include block-distance;
+  }
+
+  &__rating {
+    @include space-between-content;
+
+    &-text {
+      text-align: right;
+      font-size: $secondaryElementsSize;
+      line-height: $secondaryLineHeight;
+    }
+  }
+
+  &__pricing {
+    @include space-between-content;
+
+    line-height: $mainLineHeight;
+
+    &-text {
+      font-size: $mainElementsSize;
+      line-height: $mainLineHeight;
+      color: $secondaryColour;
+    }
+
+    &-price {
+      text-align: right;
+      font-size: 16px;
+      font-weight: $boldFontWeight;
+      line-height: $mainLineHeight;
+    }
+  }
+
+  &__button {
+    text-decoration: none;
+    text-align: center;
+    color: #fff;
+    background-color: $buttonColour;
+    border-radius: 5px;
+    display: flex;
+    font-size: 14px;
+    font-weight: $boldFontWeight;
+    text-transform: uppercase;
+    line-height: 16px;
+    padding-block: 12px;
+    padding-inline: 69px;
```

```suggestion
    padding: 12px 0;
```
Make it after all the previous comments